astroquery/linelists/cdms/core.py

```diff
@@ -304,6 +305,13 @@ def get_species_table(self, *, catfile='partfunc.cat', use_cached=True,
         ----------
         catfile : str, name of file, default 'catdir.cat'
             The catalog file, installed locally along with the package
+        use_cached : bool, optional
+            If True, use the cached file if it exists.  If False, download the
+            file from the CDMS server and save it to the cache (if ``write`` is set).
+        write : bool, optional
```

Comment:
A more verbose arg would be nice, maybe call this `write_cached` or `refresh_cashed`?

Reply:
I'll be more verbose, but I think a different keyword is warranted... `write_new_cache`, probably.